tests/e2e/client/main.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import sys
+
+import requests
+
+
+def main(url):
+    response = requests.get(url)
+
+    print(response.text)
```

Comment:
Do we want to send this to the terminal page?  As we add more and more tests this could be additional clutter.  Could we instead do an assert?